src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -260,7 +260,7 @@ final class HomeViewController: UIViewController {
 			if row == 0 {
 				showInviteFriends()
 			} else {
-				WebPageHelper.showWebPage(from: self)
+				WebPageHelper.showWebPage(from: self, urlString: AppStrings.SafariView.targetURL)
```

Comment:
Same as before, isn't this just pointing to some gibberish, or am I misunderstanding something here?

Reply:
see comment above